packages/components/plh/parent-group/utils/rapidpro.utils.ts

```diff
@@ -86,7 +86,8 @@ function transformParentWithRapidProDataToLocalFormat(
   return {
     ...rest,
     ...parsedRapidProFields,
-    id: rapidpro_uuid,
+    // Use a combined id of the parent group id and the rapidpro_uuid to ensure uniqueness
+    id: `${parentGroupId}+${rapidpro_uuid}`,
```

Comment:
nit(non-blocking): haven't seen `+` used to indicate concatenated ids much before, but don't think it would really cause much harm. Given neither parentGroupId no rapidpro_uuid will contain special characters should still be pretty easy to split on. Alternatives could otherwise be `_` (good for files, bad if either ids would include character), or sometimes I see `::` used, e.g. `parentGroup1234::parentUUID1`, but think the `+` should still do the job fine.